jina/types/arrays/mixins/traverse.py

```diff
@@ -48,13 +48,15 @@ class TraverseMixin:
     def traverse(
         self: 'T',
         traversal_paths: str,
+        check_traversal_paths: bool = True,
```

Comment:
Why is this needed? I do not like adding this to the public interface, this should not be cared by the user